app/src/main/res/values/strings.xml

```diff
@@ -92,6 +93,7 @@
     <string name="action_view_favourites">Favourites</string>
     <string name="action_view_mutes">Muted users</string>
     <string name="action_view_blocks">Blocked users</string>
+    <string name="action_view_domain_mutes">Muted instances</string>
```

Comment:
the wording used in mastodon web is `Hidden domains`. I think we should follow that.